README.md

```
@@ -3,7 +3,7 @@ workflow definition for CANRISK
 Includes variant calling against specified set of PRS variants (genotyping) and additional checks and VCF modifications to provide an output compatible with the CANRISK tool.
 
 -------
-## Current Version: 1.3.0
+## Current Version: 1.3.1
```

Markdownlint (0.35.0)

Expected: 1; Actual: 0; Above
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)